### PR TITLE
Enh/keepalive

### PR DESCRIPTION
**Pull request by satra**

### CODE REVIEW

**Comment by satra** [comment]
@synapticarbors - pypi.io does not seem to have the full repo in it's tar.gz file?

should i be pointing to github directly for releases?

File: recipes/keepalive/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    - python
+
+test:
+  imports:
+    - keepalive
+
+about:
+  home: https://github.com/wikier/keepalive
+  license: LGPL-2.1
+  license_file: LICENSE
```
Comment (mwcraig) [inline recipes/keepalive/meta.yaml]: @satra -- since the LICENSE isn't included in the source tarball feel free to drop this line.

Comment (synapticarbors) [inline recipes/keepalive/meta.yaml]: I would just comment out the `license_file` line and add a note that the file is missing due to the project missing a `MANIFEST.in` that says that the license should be copied into the release. Then possibly raise and issue upstream with the project and update the feedstock if they resolve it.

File: recipes/keepalive/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    - python
+
+test:
+  imports:
+    - keepalive
+
+about:
+  home: https://github.com/wikier/keepalive
+  license: LGPL-2.1
+  note: 'LICENSE file is missing in source due to the project missing a MANIFEST.in to copy into the release.'
```
Comment (synapticarbors) [inline recipes/keepalive/meta.yaml]: This needs to be a comment, thus beginning with a `#`

**Comment by satra** [comment]
@synapticarbors - can we merge this so we can proceed with the other two PRs? they are all dependent on each other - and their tests fail at this point because rdflib needs sparqlwrapper which needs keepalive.

**Pull request merged**